django_tasks/backends/database/models.py

```diff
@@ -124,6 +126,30 @@ def task_result(self) -> "TaskResult[T]":
 
         return result
 
+    @property
+    def canonical(self) -> Dict:
```

Comment:
Suggestion: This might be something better to store on the `TaskResult` instead. That way, it's not specific to the DB implementation.